mmpose/datasets/pipelines/shared_transform.py

```diff
@@ -276,3 +278,123 @@ def __call__(self, results):
     def __repr__(self):
         repr_str = self.__class__.__name__ + f'(transforms={self.transforms})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class PhotoMetricDistortion(object):
```

Comment:
`PhotoMetric` -> `Photometric`

It is derived from word `photometry`

Reply:
btw ~~(object)~~

Reply:
Actually, PhotoMetricDistortion is copied from mmdet. Shall we keep the name the same?

Reply:
no

Reply:
We can raise an issue there~

Reply:
cc. @yhcao6